R/layers2traces.R

```diff
@@ -873,7 +877,11 @@ geom2trace.GeomTile <- function(data, params, p) {
 
 #' @export
 geom2trace.GeomErrorbar <- function(data, params, p) {
-  make_error(data, params, "y")
+  if (params[["flipped_aes"]]) {
```

Comment:
Is this a new ggplot2 param? I haven't noticed it before (where does it come from?) Could you provide an example of where it's relevant? Preferably, provide a very small reprex that we can include as a comment to explain why this is here

Reply:
Hi Carson. 

You are right. This param was introduced in ggplot2 3.3.0 to support [bi-directional geoms](https://www.tidyverse.org/blog/2020/03/ggplot2-3-3-0/#bi-directional-geoms-and-stats). This functionality is implemented in [has_flipped_aes, ...](https://ggplot2.tidyverse.org/reference/bidirection.html).

I haven't realized that this is a more general change. In so far my PR offers only a fix for the case of `GeomErrorbar `but no general solution. I added a comment to explain why this is here together with a small reprex.

```suggestion
# Support of bi-directional GeomErrorbar introduced with ggplot2 3.3.0:
# g <- ggplot() + geom_errorbar(aes(y = "A", xmin = 1, xmax = 2))
# ggplotly(g)
if (params[["flipped_aes"]]) { 
```